operate/qa/integration-tests/src/test/java/io/camunda/operate/ProbesTestIT.java

```diff
@@ -28,7 +30,11 @@ class ProbesTestIT {
   @Nested
   @ExtendWith(SpringExtension.class)
   @SpringBootTest(
-      classes = TestApplication.class,
+      classes = {
+        TestApplication.class,
+        UnifiedConfigurationHelper.class,
+        UnifiedConfiguration.class
```

Comment:
You can move these to `TestApplication`.

Reply:
Done